pkg/cmd/pipeline/logs.go

```diff
@@ -95,8 +95,10 @@ Show logs for given Pipeline and PipelineRun:
 	c.Flags().BoolVarP(&opts.Last, "last", "L", false, "show logs for last PipelineRun")
 	c.Flags().BoolVarP(&opts.AllSteps, "all", "a", false, "show all logs including init steps injected by tekton")
 	c.Flags().BoolVarP(&opts.Follow, "follow", "f", false, "stream live logs")
-	c.Flags().BoolVarP(&opts.Timestamps, "timestamps", "t", false, "show logs with timestamp")
+	c.Flags().BoolVarP(&opts.Timestamps, "timestamps", "", false, "show logs with timestamp")
 	c.Flags().IntVarP(&opts.Limit, "limit", "", 5, "lists number of PipelineRuns")
+	c.Flags().BoolVarP(&opts.Prefixing, "prefix", "", true, "prefix each log line with the log source (task name and step name)")
+	c.Flags().StringSliceVarP(&opts.Tasks, "task", "t", []string{}, "show logs for mentioned Tasks only")
```

Comment:
why do we need this flag? is this a related change?

Reply:
@vinamra28 Are you talking about the line no. 101 ? I added that in order to ensure that we can filter out and show logs for a specific/required task only. This functionality exists in `tkn pr logs` . Now that we are formatting the output of `tkn p logs` to look like `tkn pr logs` I felt that this functionality should also be there else the logs for both the commands might be different in some cases.